infra/notifications/src/app.js

```diff
@@ -73,17 +107,23 @@ app.use(bodyParser.json({ limit: '10mb' }))
  * For development purposes only. Disabled in production.
  */
 app.get('/', async (req, res) => {
-  let markup = `<h1>Origin Notifications</h1><h2><a href="https://github.com/OriginProtocol/origin/issues/806">Learn More</a></h2>`
-
-  if (app.get('env') === 'development') {
-    const subs = await PushSubscription.findAll()
-
-    markup += `<h3>${subs.length} Push Subscriptions</h3><ul>${subs.map(
-      s =>
-        `<li><pre style="white-space: pre-wrap;word-break: break-all">${JSON.stringify(
-          s
-        )}</pre></li>`
-    )}</ul>`
+  let markup =
+    `<h1>Origin Notifications v${process.env.npm_package_version}</h1>` +
+    `<p><a href="https://github.com/OriginProtocol/origin/issues/806">Learn More</a></p>`
+
+  try {
+    if (app.get('env') === 'development') {
```

Comment:
In other places we are using `process.env.NODE_ENV`. Should we also here?

Reply:
Agreed. Fixed in 6a6df603c50c9e382fad75267f549d4d47e96db4